web/templates/settings/add_new_custom_profile_field_form.hbs

```diff
@@ -26,11 +26,12 @@
             <input type="text" id="profile_field_hint" class="modal_text_input" name="hint" autocomplete="off" maxlength="80" />
             <div class="alert" id="admin-profile-field-hint-status"></div>
         </div>
-        <div class="input-group" id="profile_field_choices_row">
+        <div class="input-group profile-field-choices-wrapper" id="profile_field_choices_row">
             <label for="profile_field_choices" class="modal-field-label">{{t "Field choices" }}</label>
             <table class="profile_field_choices_table">
                 <tbody id="profile_field_choices" class="profile-field-choices"></tbody>
             </table>
+            <button disabled type="button" class="button white rounded alphabetize-choices-button">{{t "Alphabetize choices" }}</button>
```

Comment:
Why disable the button when `maybe_toggle_alphabetize_button` is called upon clicking the edit button of the profile field?


Also, the commit message should discuss changes in the commit but you maybe accidentally write "PR"?

Reply:
Oh okay got it.